Sources/XiEditor/AppDelegate.swift

```diff
@@ -169,6 +151,8 @@ class AppDelegate: NSObject, NSApplicationDelegate {
         self.xiCore = xiCore
         updateRpcTracingConfig(collectSamplesOnBoot)
 
+        // Setups config folder and file
```

Comment:
Total 100% nit: I think this function is well enough named that the comment is redundant.